### ISSUE TITLE
Framerate feels choppy

### ISSUE THREAD

**Issue by BillMess** [osxmidi/LinVst#65]
Every plugin I try with LinVst, the framerate of the GUI feels choppy. However, when I run the same plugin inside of Airwave, it feels butter smooth. Do you have any ideas what could cause this? You would think that LinVst should perform better since it is newer.

**Comment by osxmidi** [comment]
I've never heard about that problem before and that includes LinVst being used on multi distros.

If it was a common problem then I would have heard about it by now.

I would guess that it's some sort of hardware/wine/library incompatibility problem.

What window manager do you have

wmctrl -m

sudo apt-get install wmctrl

**Comment by BillMess** [comment]
I'm running Linux Mint 19, this is the output of wmctrl -m:
```
Name: Mutter (Muffin)
Class: N/A
PID: N/A
Window manager's "showing the desktop" mode: N/A
```
I've been having graphics issues with my system all over the place. I posted before that I was having trouble getting Serum to run smoothly. But also programs like Blender feel super laggy unless I run it with "env LIBGL_DRI3_DISABLE=1 blender". And Firefox sometimes glitches out while scrolling. Even in Windows 10 with the latest drivers some games don't perform well.

I have an AMD R9 285 graphics card. I've tried putting a friend's RX480 in my system, but it was still the same. I've also tried removing the card and running off of the processor's graphics (i7-4790k).

Something I think I'll try next is downgrading the BIOS firmware on my motherboard (ASUS Z97-A) and see if that fixes the issue. I'm running the latest one for my motherboard.
